dxr/plugins/rust/tests/test_data/code/makefile

```diff
@@ -0,0 +1,6 @@
+all: code
```

Comment:
You could use SingleFileTestCase if you didn't want to have all this boilerplate (makefile, dxr.config.in, etc.); we just made the source file name customizable so it doesn't assume "main.cpp". But I assume you're finding it convenient to be able to build and examine in place for now.